vnext/Microsoft.ReactNative/Modules/AppStateModule.cpp

```diff
@@ -40,6 +44,20 @@ void AppState::Initialize(winrt::Microsoft::ReactNative::ReactContext const &rea
                 strongThis->SetActive(true);
               }
             });
+
+        m_activatedEventRevoker = window.Activated(
```

Comment:
I wonder if perhaps we should just always use window.Activated and not Application.EnteredBackground/LeavingBackground to mean "active".
This is somewhat loosely defined in RN since the notion of "active" in mobile and in desktop is different.
@stmoy any thoughts on what "active" should mean on desktop?

Reply:
@asklar yes indeed, I believe we don't need the "EnteredBackgound" and "LeavingBackground" transitions at all as the "Focused" and "Unfocused" states covers those situations as well.

Reply:
@asklar I feel like we should go ahead and remove EnteredBackground/LeavingBackground. @stmoy are you able to give your perspective?

Reply:
Aligning to Window.Activated makes sense to me - I wouldn't want to reinvent the wheel and redefine what "active" means for desktop.

Reply:
I updated the PR description with the final approach we went with